test/integration/smart_answer_flows/marriage_abroad_test.rb

```diff
@@ -886,9 +884,7 @@ def self.translations
   context "ceremony in Turkey, resident in Turkey, partner other" do
```

Comment:
It's a minor thing, but the title of this test should probably be changed to reflect that it's testing the same sex route, and the test above does with opposite sex.